### ISSUE TITLE
Wcag-aaa correspondance

### ISSUE THREAD

**Issue by LaZay** [w3c/a11y-discov-vocab#59]
In the [1. EPUB and ONIX](https://w3c.github.io/a11y-discov-vocab/crosswalk/#epub-onix) section, the following correspondance rule is missing.
`dcterms:conformsTo with the URL [dcterms:conformsTo with the URL http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-aa](http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-aaa)`

So far, there is no ONIX corresponding value in the [code list 196](https://ns.editeur.org/onix/en/196). Which must be pointed out , so it can evolve in the future.

**Comment by chrisONIX** [comment]
@LaZay 
There is a value in ONIX for WCAG level AA (code 85) and WCAG level AAA (code 86) 
They have also been mapped in the latest crosswalk - https://w3c.github.io/a11y-discov-vocab/crosswalk/

**Comment by chrisONIX** [comment]
dcterms:conformsTo with the URL `http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-aaa

This is not in the crosswalk - does this need to be added - EPUB Accessibility 1.0 and WCAG AAA?

**Comment by clapierre** [comment]
@chrisONIX Since ONIX does not define EPUB Accessibility 1.0 WCAG 2.0 AAA
How does this sound?
First use 196-03 for EPUB Accessibility 1.0 WCAG 2.0 AA (This will at least provide that it conforms to the EPUB Accessibility 1.0 standard at the AA level which technically it is, but has gone beyond that.

Then override the WCAG level by stating AAA (code 86)
We could also do WCAG 2.0 with code (code 80) 

What do you all think?
Include Code 03 for inclusion of EPUB 1.0?
or just put under ONIX codes 80 & 86? but if we do this then its saying it only conforms to WCAG 2.0 AA not the EPUB Accessibility 1.0 Specification as well.

Also, what codes go under:
EPUB to MARC21
ONIX to MARC21 using EPUB Vocabulary
and
UNIMARC

**Comment by chrisONIX** [comment]
@mattgarrish I would be curious to see if anyone did meet AAA for 1.0 - we reserved a space for it for a long time, but no one ever asked us to add it.

@clapierre if we add 1.0 , WCAG 2.0 , AAA to crosswalk I think this is the best for now (if any publisher actually met AAA for 1.0 we could add an ONIX code?

First use 196-03 for EPUB Accessibility 1.0 WCAG 2.0 AA (This will at least provide that it conforms to the EPUB Accessibility 1.0 standard at the AA level which technically it is, but has gone beyond that.

Then override the WCAG level by stating AAA (code 86)

**Comment by clapierre** [comment]
I agree with you @mattgarrish we are just adding it since the original 1.0 did have a AAA option. I also don't think anyone ever used it. So this is just for crossing t's at this point.
